content/en/guide/v10/typescript.md

```diff
@@ -92,7 +93,7 @@ function Greeting({ name = "User" }: GreetingProps) {
 }
 ```
 
-Preact also ships a `FunctionComponent` type to annotate anonymous functions. `FunctionComponent` also adds a type for `children`:
```

Comment:
`FunctionComponent` type corresponds all functional components, not only an anonymous function.
This sentence is confusing.